pq-crypto/sike_r2/config.h

```diff
@@ -1,232 +1,228 @@
-    (shiftOut) = __shiftleft128((lowIn), (highIn), (shift));
-
-// 64x64-bit multiplication
-#define MUL128(multiplier, multiplicand, product) \
-    (product)[0] = _umul128((multiplier), (multiplicand), &(product)[1]);
-
-// 128-bit addition with output carry
-#define ADC128(addend1, addend2, carry, addition)                           \
-    (carry) = _addcarry_u64(0, (addend1)[0], (addend2)[0], &(addition)[0]); \
-    (carry) = _addcarry_u64((carry), (addend1)[1], (addend2)[1], &(addition)[1]);
-
-#define MULADD128(multiplier, multiplicand, addend, carry, result) \
-    ;                                                              \
-    {                                                              \
-        uint128_t product;                                         \
-        MUL128(multiplier, multiplicand, product);                 \
-        ADC128(addend, product, carry, result);                    \
-    }
-
-#elif ((TARGET == TARGET_AMD64 || TARGET == TARGET_ARM64) && OS_TARGET == OS_LINUX)
-
-// Digit multiplication
-#define MUL(multiplier, multiplicand, hi, lo)                                    \
-    {                                                                            \
-        uint128_t tempReg = (uint128_t)(multiplier) * (uint128_t)(multiplicand); \
-        *(hi) = (digit_t)(tempReg >> RADIX);                                     \
-        (lo) = (digit_t) tempReg;                                                \
-    }
-
-// Digit addition with carry
-#define ADDC(carryIn, addend1, addend2, carryOut, sumOut)                                       \
-    {                                                                                           \
-        uint128_t tempReg = (uint128_t)(addend1) + (uint128_t)(addend2) + (uint128_t)(carryIn); \
-        (carryOut) = (digit_t)(tempReg >> RADIX);                                               \
-        (sumOut) = (digit_t) tempReg;                                                           \
-    }
-
-// Digit subtraction with borrow
-#define SUBC(borrowIn, minuend, subtrahend, borrowOut, differenceOut)                               \
-    {                                                                                               \
-        uint128_t tempReg = (uint128_t)(minuend) - (uint128_t)(subtrahend) - (uint128_t)(borrowIn); \
-        (borrowOut) = (digit_t)(tempReg >> (sizeof(uint128_t) * 8 - 1));                            \
-        (differenceOut) = (digit_t) tempReg;                                                        \
-    }
-
-// Digit shift right
-#define SHIFTR(highIn, lowIn, shift, shiftOut, DigitSize) \
-    (shiftOut) = ((lowIn) >> (shift)) ^ ((highIn) << (RADIX - (shift)));
-
-// Digit shift left
-#define SHIFTL(highIn, lowIn, shift, shiftOut, DigitSize) \
-    (shiftOut) = ((highIn) << (shift)) ^ ((lowIn) >> (RADIX - (shift)));
-
-#endif
-
-#endif
+/********************************************************************************************
```

Comment:
CLion reformatted the line endings on this file on this file (converted Windows `\r\n` to *nix `\n`), so git detects that the entire file has changed. I've commented below on the specific areas that actually have non-whitespace changes.